### PR TITLE
updateClaim function added

### PR DESCRIPTION
**Pull request by Setz0r**
updateClaim LUA function added to make a monster claimed through script.
similar to updateEnmity but actually marks the monster as claimed.  good for scripted spawning of quest monsters or fished up NM

example in LUA:

SpawnMob(monsterid):updateClaim(player);

### CODE REVIEW

**Comment by maxtherabbit** [comment]
there are unrelated changes in both files that need to be cleaned up

File: src/map/lua/lua_baseentity.cpp [hunk 1]
```diff
@@ -9111,10 +9135,10 @@ inline int32 CLuaBaseEntity::getPartyLeader(lua_State* L) // Todo: also add abil
             if (PLeaderName != nullptr)
             {
                 lua_pushstring(L, PLeaderName);
-                return 1;
-            }
+            return 1;
```
Comment (teschnei) [inline src/map/lua/lua_baseentity.cpp]: ?

**Comment by teschnei** [comment]
replace your tabs with spaces

**Comment by Setz0r** [comment]
I updated the pull request to convert all tabs to spaces and remove unneeded code.

**Pull request merged**